web/template/layout/edit-panel.gotmpl

```diff
@@ -16,48 +16,36 @@
           <h3 class="govuk-heading-s govuk-!-margin-0">Edit tasks</h3>
         </div>
       </header>
-          <div class="govuk-width-container">
-          <form action="{{ prefix "/" }}" method="POST" id="reassign-task-form" class="ignore-pagination-bottom">
```

Comment:
Looks like you've removed the `<form>` element from here, but not recreated it anywhere else? Just checking that's intentional. Is it in the parent template somewhere?

Reply:
Yeah the reason it was breaking was the additional form element, most of the workflow is submitted as a single page form and putting a second form within this caused issues.

Reply:
Cool, thought it might be, but as it wasn't part of the PR wanted to check. :)